pkg/apis/operator/v1alpha1/tektonconfig_types.go

```diff
@@ -49,6 +49,12 @@ func (tp *TektonConfig) GetStatus() TektonComponentStatus {
 
 // Prune defines the pruner
 type Prune struct {
+	// image used in cron job
+	// +optional
+	Image string `json:"image,omitempty"`
```

Comment:
This feature is to replace the tkn image, with user-defined image in config CR, do we need this cc @vdemeester

Reply:
@vdemeester could you please provide your thoughts on this?

Reply:
@vdemeester I am going to remove this image option, we can manage it via environment variable.
* we can update it via operator deployment environment
* in olm managed version, we can update the environment in subscription cr